packages/entity-list/src/components/ColumnPicker/ColumnPicker.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types'
+import React, {useState} from 'react'
+import {Button, SearchBox, Typography} from 'tocco-ui'
+import styled from 'styled-components'
+import {intlShape} from 'react-intl'
+
+const StyledUl = styled('ul')`
+  && {
+    list-style-type: none;
+  }
+`
+
+const StyledCheckbox = styled('input')`
+  && {
+    vertical-align: top;
+  }
+`
+
+const ColumnPicker = ({columns, onOk, intl}) => {
+  const [selectedColumns, setSelectedColumns] = useState({})
+  const [searchTerm, setSearchTerm] = useState(null)
+  const items = columns
```

Comment:
you could create the "useMemo" hook. Then items would not be calculated on every rerender.